builder/qemu/step_type_boot_command.go

```diff
@@ -154,6 +155,19 @@ func vncSendString(c *vnc.ClientConn, original string) {
 			continue
 		}
 
+		if strings.HasPrefix(original, "<wait") && strings.HasSuffix(original, ">") {
+			re := regexp.MustCompile(`<wait([0-9a-z]+)>$`)
```

Comment:
For duration I think a-z is too broad. Do you want `[0-9hms]`? What happens with `1m 30s`? It might be easier if this is just an int value in seconds.